src/registrar/assets/sass/_theme/_admin.scss

```diff
@@ -525,17 +525,25 @@ address.dja-address-contact-list {
 }
 
 // Collapse button styles for fieldsets
-.module.collapse {
+.module.collapse--dotgov {
     margin-top: -35px;
     padding-top: 0;
     border: none;
-    h2 {
+    button {
         background: none;
-        color: var(--body-fg)!important;
         text-transform: none;
-    }
-    a {
         color: var(--link-fg);
+        margin-top: 8px;
+        margin-left: 10px;
+        span {
+            text-decoration: underline;
+        }
+    }
+}
+.collapse--dotgov.collapsed .collapse-toggle--dotgov  {
+    display: inline-block!important;
```

Comment:
We're working with USWDS and DJANGO styles, so a lot of times an easy solution is !important. IMO, it's good to avoid important as it can lead to unexpected overrides elsewhere in your stylesheet, but note the selector: You have 2 custom classes that are obviously single use, so there's no risk in this case.

Reply:
Got it, thank you for explaining!